db/migrate/20180405090900_create_news.rb

```diff
@@ -0,0 +1,11 @@
+class CreateNews < ActiveRecord::Migration[5.1]
+  def change
+    create_table :news do |t|
+      t.string :title
+      t.text :description
+      t.integer :news, :user_id
```

Comment:
did you run the migration?

Reply:
@okohuch yes

Reply:
and it passes successfully?

Reply:
do you see that something is wrong with this line?

Reply:
@okohuch it passed successfully. maybe delete :news in user_id creating?